src/assets/githubAction/githubActionBeginner.md

```diff
@@ -0,0 +1,130 @@
+---
+title: GitHub Actions - CI & CD
+authorName: Nabin Ale
+authorAvatar : https://avatars.githubusercontent.com/u/61624650?s=400&u=8f992a9ca189af19c40cb070a5af1e02669d3884&v=4
+authorLink: https://github.com/nabim777
+createdAt: Oct 30, 2023
+tags: CI, CD, github, githubAction
+banner: https://blog.jankaritech.com/src/assets/githubAction/images/githubaction_banner.png
+---
+GitHub Actions is an automation and CI/CD service provided by GitHub. It allows us to automate our software development process or workflows, from building and testing code to deploying it to different environments. It's a really effective tool that can both increase software quality and save us time.
```

Comment:
```suggestion
GitHub Actions is an automation and CI/CD service provided by GitHub. It allows us to automate our software development process or workflows, from building and testing code to deploying it to different environments. It's a really an effective tool that can both increase software quality and save our time.
```

Reply:
I think there are two ways how the last sentence can be expressed in English, either how Nabin first proposed it:

It's a really effective tool
--> meaning very effective

or 

It's really an effective tool
--> meaning it is true that this is an effective tool